plugins/com.gwtplugins.gwt.eclipse.core/src/com/google/gwt/eclipse/core/modules/AbstractModule.java

```diff
@@ -124,13 +119,7 @@ private static List<String> getElementsAttributes(Document doc, String elementNa
     return attrValues;
   }
 
-  protected final IStorage storage;
-
-  private String qualifiedName;
-
-  protected AbstractModule(IStorage storage) {
-    assert (storage != null);
-    this.storage = storage;
+  protected AbstractModule() {
```

Comment:
shouldnt even be necessary, the class is already abstract and has no other ctors

Reply:
Ok